integration-test/langfuse-integration-langchain.spec.ts

```diff
@@ -14,326 +14,345 @@ const { ConversationChain, LLMChain, createExtractionChainFromZod } = require(`$
 import { z } from "zod";
 
 import { Langfuse, CallbackHandler } from "../langfuse-langchain";
-import { LF_HOST, LF_PUBLIC_KEY, LF_SECRET_KEY, getTraces } from "./integration-utils";
+import { LANGFUSE_BASEURL, LANGFUSE_PUBLIC_KEY, getHeaders, getTraces } from "./integration-utils";
 
-describe("simple chains", () => {
+describe("Langchain", () => {
   jest.setTimeout(30_000);
   jest.useRealTimers();
 
-  it("should execute simple llm call", async () => {
-    const handler = new CallbackHandler({
-      publicKey: LF_PUBLIC_KEY,
-      secretKey: LF_SECRET_KEY,
-      baseUrl: LF_HOST,
-      sessionId: "test-session",
+  describe("setup", () => {
+    it("instantiates with env variables", async () => {
+      const callback = new CallbackHandler();
+      // @ts-expect-error
```

Comment:
We could change getFetchOptions from private to public with underscore for testing use?